apps/explorer/src/ui/Banner.tsx

```diff
@@ -29,27 +33,70 @@ const bannerStyles = cva(
                 md: 'px-3 py-2',
                 lg: 'p-5',
             },
+            shadow: {
+                true: 'shadow-md',
+            },
+            border: {
+                true: '',
+            },
         },
         defaultVariants: {
             variant: 'message',
             spacing: 'md',
+            border: false,
         },
+        compoundVariants: [
```

Comment:
This is a bit customized, but I do not think we have all the exact hex values for these colors

Reply:
Can you work with @mystie711 to get these incorporated into the theme?